membership-attribute-service/app/components/TouchpointComponents.scala

```diff
@@ -168,14 +162,13 @@ class TouchpointComponents(
     audience = Some(OktaAudience(conf.getString("okta.verifier.audience"))),
     clientId = None,
   )
-  lazy val identityPlayAuthService: IdentityPlayAuthService[UserFromToken, DefaultIdentityClaims] = {
+  lazy val identityPlayAuthService: IdentityPlayAuthService = {
```

Comment:
most of the logic changes are due to this which came in a preceding version of identity-auth-play that we hadn't picked up yet